distribution/pom.xml

```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Oh, so it's not limited to code, it also gets applied to poms. Nice.

Reply:
There's also this "pom" section that can be configured: https://github.com/diffplug/spotless/tree/main/plugin-maven#maven-pom where it can keep the pom "formatted" too, but I didn't want to try it as we have some cases where the profile has to go last (those reporting modules etc....) so I thought better leave it as is 😃

Reply:
Ah, so this one was manual. Gotcha.

You're right, it's probably safer to skip poms, I didn't think of that.